tests/unit/airlock/test_views.py

```diff
@@ -281,6 +281,53 @@ def test_api_airlock_event_error(api_rf, event_type, updates, error):
     }
 
 
+@patch("airlock.views._get_github_api", FakeGitHubAPIWithErrors)
+def test_api_airlock_event_multiple_errors(api_rf, slack_messages):
+    author = UserFactory()
+    user = UserFactory()
+    WorkspaceFactory(name="test-workspace")
+    backend = BackendFactory(auth_token="test", name="test-backend")
+    BackendMembershipFactory(backend=backend, user=user)
+
+    # A request approved event sends 3 notifications:
+    # - sends an email that includes the release URL,
+    # - updates the github issue
+    # - posts to slack
+    # This notification attempt will error on sending the email, because
+    # we haven't created a Release for it, and it'll error creating the
+    # GitHub issue update because we've mocked it that way. All events will
+    # be attempted, and the two errors will be reported in the response
```

Comment:
It feels slightly brittle to have this test depend on us not setting up the database in the way that we would expect to find it in production.  Is it feasible to mock both errors?

Reply:
yes